### PR TITLE
fix: Moved instruction to proper place and removed assert text in Font Awesome challenges

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-all-of-our-buttons.english.md [hunk 1]
```diff
@@ -10,13 +10,12 @@ challengeType: 0
...
 Font Awesome is a convenient library of icons. These icons are vector graphics, stored in the <code>.svg</code> file format. These icons are treated just like fonts. You can specify their size using pixels, and they will assume the font size of their parent HTML elements.
-Use Font Awesome to add an <code>info-circle</code> icon to your info button and a <code>trash</code> icon to your delete button.
-Note: The <code>span</code> element is an acceptable alternative to the <code>i</code> element for the directions below.
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+Use Font Awesome to add an <code>info-circle</code> icon to your info button and a <code>trash</code> icon to your delete button.
+Note: The <code>span</code> element is an acceptable alternative to the <code>i</code> element for the directions below.
```
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-all-of-our-buttons.english.md]:
```suggestion

<strong>Note:</strong> The <code>span</code> element is an acceptable alternative to the <code>i</code> element for the directions below.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my suggestion and also need to fix merge conflicts.

**Comment by thecodingaviator** [comment]
@RandellDawson Changes made

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator OK, LGTM.

**Comment by ojeytonwilliams** [comment]
@thecodingaviator Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Pull request merged**